sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -258,7 +258,9 @@ case class AlterViewAsCommand(
   def markAsAnalyzed(): LogicalPlan = copy(isAnalyzed = true)
 
   override def run(session: SparkSession): Seq[Row] = {
-    if (session.sessionState.catalog.isTempView(name)) {
+    val isTemporary = session.sessionState.catalog.isTempView(name)
+    verifyTemporaryObjectsNotExists(session.sessionState.catalog, isTemporary, name, query)
```

Comment:
there is a `verifyAutoGeneratedAliasesNotExists` in the latest master code. could you put them together?

Reply:
OK